transport/nio/src/main/java/org/kaazing/gateway/transport/nio/internal/NioIdleFilter.java

```diff
@@ -1,63 +0,0 @@
-/**
```

Comment:
We will need these changes in until you are feature complete for your whole story.... Sorry